utils/dataloaders.py

```diff
@@ -163,8 +189,68 @@ def __len__(self):
     def __getitem__(self, idx):
         sample_path = self.img_paths[idx]
         sample = imread(sample_path)
+        #print(sample_path)
+        if self.transform:
+            sample = self.transform(sample)
+        # Since there are no labels, we just return 0 for the "label" here
+        return sample, 0
+
+class ChairsDataset(Dataset):
+    """CelebA dataset with 64 by 64 images."""
 
+    def __init__(self, path_to_data, subsample=1, transform=None):
+        """
+        Parameters
+        ----------
+        subsample : int
+            Only load every |subsample| number of images.
+        """
+        #print(path_to_data)
+        #self.img_paths = glob.glob(path_to_data + '/images_64/*')[::subsample]
+        self.img_paths = glob.glob(path_to_data + '/*')[::subsample]
+        self.transform = transform
+    def __len__(self):
+        return len(self.img_paths)
+
+    def __getitem__(self, idx):
+        sample_path = self.img_paths[idx]
+        sample = imread(sample_path)
+        #sample = self.loader(path)
+        #sample = F.to_pil_image(sample)
         if self.transform:
             sample = self.transform(sample)
         # Since there are no labels, we just return 0 for the "label" here
         return sample, 0
+
+def resize_images_celeba():
+    directory_in = os.path.join(os.path.dirname(DIR), 'data', 'celeba', 'img_align_celeba')
+    directory_out = os.path.join(os.path.dirname(DIR), 'data', 'celeba', 'img_align_celeba_64')
+    i=0
+    for file_name in os.listdir(directory_in):
+        if (i%100)==0:
+            print("iteration nr: %d / %d" %(i,len(os.listdir(directory_in))))
```

Comment:
again use `logging`. And better messages, i.e. say that you are resizing `images_celeba`. Also use `.format()` instead of `%` for replacing string